docs/commands/highlight.md

```diff
@@ -0,0 +1,93 @@
+## Command highlight ##
+
+This command sets up custom highlighting for user set strings.
+
+The syntax is as following:
+
+```
+highlight (list|del|add|clear|regex) [match] [color]
+```
+
+This command provides the following aliases:
+
+  - `hh`
```

Comment:
Why two two-letter aliases? I'd stick with `hl`, or is `hh` the same as used in another tool?